src/fides/api/service/saas_request/override_implementations/appsflyer_request_overrides.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any, Dict, List
+
+from fides.api.graph.traversal import TraversalNode
+from fides.api.models.policy import Policy
+from fides.api.models.privacy_request import PrivacyRequest
+from fides.api.service.connectors.saas.authenticated_client import AuthenticatedClient
+from fides.api.service.saas_request.saas_request_override_factory import (
+    SaaSRequestType,
+    register,
+)
+from fides.api.util.collection_util import Row
+
+
+@register("appsflyer_user_read", [SaaSRequestType.READ])
+def appsflyer_user_read(
+    client: AuthenticatedClient,
+    node: TraversalNode,
+    policy: Policy,
+    privacy_request: PrivacyRequest,
+    input_data: Dict[str, List[Any]],
+    secrets: Dict[str, Any],
+) -> List[Row]:
+    """
+    Gather the full list of applications set up, and retrieve the application name and
+    platform information for use with erasure endpoint.
+    """
+
+    user_ids = input_data.get("user_id")  #  lists
+    app_ids = input_data.get("app_id")  #  lists
+    # import pdb
+
+    # pdb.set_trace()
```

Comment:
Remove all the commented out `pdb.set_trace()` once this is ready for review

Reply:
Added my saas_config.yml for AppsFlyer to 1password ([here](https://share.1password.com/s#rGCDpSJcKONHjushoxFqP4W0qsEJXMI0W3qFTSeYcu8)), as well as the account information in a [separate entry](https://share.1password.com/s#oUrPNpye257COix7RfNgr4Zwc44E7i3l4HtttC5kKGU), and lastly the API token ([here](https://share.1password.com/s#4Y6XXNB6dDleRiyTW2T-tPIz2RPyygHQpHPag3l_YuE)).